engine/runner/src/main/java/org/enso/runner/JavaFinder.java

```diff
@@ -68,7 +69,10 @@ private static boolean isOnWindows() {
    */
   private static Path findJavaExecutableInDistributionRuntimes() {
     var env = new Environment() {};
-    var distributionManager = new DistributionManager(env);
+    var distributionManager = new PortableDistributionManager(env);
```

Comment:
There should not be a `PortableDistributionManager`. The `DistributionManager` itself should decide whether it deals with the portable distribution or not.

Reply:
I know it's kind of unrelated, just quite surprised that the `DistributionManager` can't do that.